src/main/webapp/index.html

```diff
@@ -100,8 +106,93 @@ <h1>Favorite SDKs</h1>
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+
+    <!-- Initialize firebase products -->
+    <script src="https://www.gstatic.com/firebasejs/7.15.5/firebase-app.js"></script>
+    <script defer src="https://www.gstatic.com/firebasejs/7.15.5/firebase-auth.js"></script>
+
+    <script>
+    // Your web app's Firebase configuration
+        var firebaseConfig = {
+            apiKey: "AIzaSyAGk_4TZiVf_TslWON5hNQDYVODuJifP7A",
+            authDomain: "com-sdk-dashboard-step-2020.firebaseapp.com",
+            databaseURL: "https://com-sdk-dashboard-step-2020.firebaseio.com",
+            projectId: "google.com:sdk-dashboard-step-2020",
+            messagingSenderId: "412575630040",
+            appId: "1:412575630040:web:824b2d31f4e1f06b100dad",
+        };
+
+        // Initialize Firebase
+        firebase.initializeApp(firebaseConfig);
+
+        // Redirect to Google Sign In
+        function googleSignIn() {
+            firebase.auth().onAuthStateChanged(function(user) {
+            if (user) {
+                // User is signed in.
+            } else {
+                // No user is signed in.
+                var provider = new firebase.auth.GoogleAuthProvider();
+                firebase.auth().signInWithRedirect(provider);
+                firebase.auth().getRedirectResult().then(function(result) {
+                    if (result.credential) {
+                        // This gives you a Google Access Token. You can use it to access the Google API.
+                        var token = result.credential.accessToken;
```

Comment:
We don't seem to be using this access token for anything. Why do we store this? If this is something you plan on using in future, you can add a TODO instead of this code.

Reply:
I stored it initially because it was advised when I was looking up how to do the authentication

Reply:
I didn't know then but I don't think we are going to use it